api.go

```diff
@@ -492,8 +492,8 @@ func (d *Context) GetLength(index int) int {
 }
 
 // See: http://duktape.org/api.html#duk_get_lstring
-func (d *Context) GetLstring(index int, outLen int) string {
-	if s := C.duk_get_lstring(d.duk_context, C.duk_idx_t(index), (*C.duk_size_t)(unsafe.Pointer(&outLen))); s != nil {
+func (d *Context) GetLstring(index int, outLen *int) string {
```

Comment:
I understand what you are saying, your english is totally fine. I think it makes sense to update the api now, especially after looking at duktapes docs again. What about updating the apis to something like below:

```go
func (d *Context) GetLstring(index int, outLen int) (string, int) {
  if s := C.duk_get_lstring(d.duk_context, C.duk_idx_t(index), (*C.duk_size_t)(unsafe.Pointer(&outLen))); s != nil {
    return C.GoString(s), outLen
  }
  
  return "", 0
}
```

This feels more like idiomatic go to me, and it makes the code a little bit cleaner, and easier for go-duktape users to update their code. For example, [example_byte_code.go](https://github.com/mosolovsa/go-duktape/blob/BytecodeSerialization/example_bytecode_test.go#L37-L43) could be:
```go
// void *duk_get_buffer(duk_context *ctx, duk_idx_t idx, duk_size_t *out_size);
// typedef size_t duk_size_t;
// duk_size_t equivalent in Go is uint, however slice len is of a type int
// and I don't think that the limit of max(int) can be reached so for the sake of simplicity
// lets assume that bytecode size is always can be contained in int
rawmem, sz := ctxSerialize.GetBuffer(-1, 0)
```
Thoughts? @mosolovsa @olebedev Sorry about all the back and forth, but depending on what you guys think this should be the last issue. Thanks for the pull request!